app/page/template/content/conversation/input-bar.htm

```diff
@@ -29,7 +29,7 @@
                   data-uie-name="input-message"
                   dir="auto">
         </textarea>
-        <div class="shadow-input" data-bind="html: richTextInput, scrollSync: '#conversation-input-bar-text', resize: input" data-uie-name="input-message-rich-text"></div>
+        <div class="shadow-input" data-bind="html: richTextInput, scrollSync: '#conversation-input-bar-text', resize: input, resize_callback: scrollMessageList" data-uie-name="input-message-rich-text"></div>
```

Comment:
`resize_callback` -> `resizeCallback` if possible

Reply:
Then we have to rename in other places as well. This was missed during the renaming of binding and is decared in `CommonBindings.js`